apps/grc-platform/backend/internal/scim/client_test.go

```
@@ -87,6 +87,66 @@ func TestListUsersByDomain_ShortPage(t *testing.T) {
 	}
 }
 
+// TestNewExternalClient_HitsExternalOrgPath confirms NewExternalClient's
+// requests target organizations/external/*, not organizations/internal/* —
+// the two orgs hold genuinely distinct identities (see the design doc's
+// §1.4), so a client wired to the wrong path would silently never resolve
+// anyone, or worse, resolve the wrong org's uuid space.
+func TestNewExternalClient_HitsExternalOrgPath(t *testing.T) {
+	tokenSrv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		json.NewEncoder(w).Encode(tokenResponse{AccessToken: "test-token", ExpiresIn: 3600})
```

golangci-lint (2.12.2)

[error] Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)